batch/batch/worker/worker.py

```diff
@@ -345,11 +347,12 @@ def __init__(self, task_manager: aiotools.BackgroundTaskManager):
         self.internet_interface = INTERNET_INTERFACE
 
     async def reserve(self):
-        for subnet_index in range(N_SLOTS):
+        for subnet_index in range(N_SLOTS + N_JVM_CONTAINERS):
             public = NetworkNamespace(subnet_index, private=False, internet_interface=self.internet_interface)
             await public.init()
             self.public_networks.put_nowait(public)
 
+        for subnet_index in range(N_SLOTS):
```

Comment:
b/c JVMJobs can't be on the private network?

Reply:
Yes I don't see a reason for JVMJobs to use the private network. Private basically just means you can talk to the metadata server.